Several versions of llvm can coexist in the archive (at the time of this
writing, 2.9, 3.0, 3.1 and 3.2), so pick a version. Let's assume the last version
receives bug fixes quickly, so pick 3.2 for now.

--- a/configure.ac
+++ b/configure.ac
@@ -1638,9 +1638,9 @@ if test "x$enable_gallium_llvm" = xauto;
 fi
 if test "x$enable_gallium_llvm" = xyes; then
     if test "x$llvm_prefix" != x; then
-        AC_PATH_PROG([LLVM_CONFIG], [llvm-config], [no], ["$llvm_prefix/bin"])
+        AC_PATH_PROG([LLVM_CONFIG], [llvm-config-3.2], [no], ["$llvm_prefix/bin"])
     else
-        AC_PATH_PROG([LLVM_CONFIG], [llvm-config], [no])
+        AC_PATH_PROG([LLVM_CONFIG], [llvm-config-3.2], [no])
     fi
 
     if test "x$LLVM_CONFIG" != xno; then
